src/web/mock_adm_response1.json

```diff
@@ -23,6 +23,14 @@
             "image_url": "https://cdn.example.com/805.jpg",
             "advertiser_url": "https://www.lph-nm.biz/?utm_source=google&utm_campaign=quux*{keyword}_m*{match-type}_d*BARBAZ_22",
             "impression_url": "https://example.net/static?id=ABAD1DEA"
+        },
+        {
+            "id": 907,
+            "name": "Lasagna Come Out Tomorrow",
```

Comment:
Lol. HERE WE GO, the Pun Master strikes again!

Reply:
😹 Now we have The Office, Breaking Bad, and The Good Place in our fixtures!